Code/autopkglib/URLDownloader.py

```diff
@@ -69,6 +69,19 @@ class URLDownloader(URLGetter):
                 "'prefetch_filename' is useful for URLs with redirects."
             ),
         },
+        "external_metadata": {
```

Comment:
The URLDownloader input variable is `external_metadata`, but the URLGetter changes still reference `external_metadata_path`. Is there a discrepancy that needs fixing here?

Reply:
That was a PEBCAK

Reply:
That was a PEBCAK